projects/Makefile

```diff
@@ -288,21 +289,19 @@ BITTBOYFILES    :=  GPSDLMain.o \
 					BITTBOYMidiService.o \
 					SDLGUIWindowImp.o
 
-MIYOOFILES    :=  GPSDLMain.o \
+MIYOOFILES := GPSDLMain.o \
 					UnixFileSystem.o \
-					DINGOOAudio.o \
+					SDLAudio.o \
```

Comment:
This might be great for other platforms too actually, pulseaudio hasn't given me any love tbh.